h2o-core/src/main/java/water/rapids/BinaryMerge.java

```diff
@@ -209,7 +210,8 @@ public void compute2() {
     // occur, they only happen for leftMSB 0 and 255, and will quickly resolve
     // to no match in the right bucket via bmerge
     t0 = System.nanoTime();
-    bmerge_r(_leftFrom, leftTo, -1, rightN);
+    binaryMergeOneMSB(_leftFrom, leftTo, -1, rightN);
+  //  bmerge_r(_leftFrom, leftTo, -1, rightN);  // search for match row in rite frame to left value
```

Comment:
delete commented out code

Reply:
I want to keep that old code here until the timing test is completed.

Reply:
I removed the code